src/web/components/error/errormessage.jsx

```diff
@@ -22,7 +22,8 @@ import {isDefined} from 'gmp/utils/identity';
 import Divider from 'web/components/layout/divider';
 import Layout from 'web/components/layout/layout';
 
-import StNonAvailableIcon from 'web/components/icon/stnonavailableicon';
+import {AlertCircle} from 'lucide-react';
```

Comment:
Is 'lucide-react' already a direct dependency of GSA?

Reply:
I think through the opensight UI lib.

Reply:
IMHO dependencies used in imports should be added as direct dependencies in the package.json. If it is added only via opensight ui it is a indirect dependency.